packages/cli/src/commands/server/server.ts

```diff
@@ -22,60 +21,6 @@ export default {
       name: '--host [string]',
       default: '',
     },
-    {
-      name: '--projectRoot [path]',
-      description: 'Path to a custom project root',
-      parse: (val: string) => path.resolve(val),
-    },
-    {
-      name: '--watchFolders [list]',
-      description:
-        'Specify any additional folders to be added to the watch list',
-      parse: (val: string) =>
-        val.split(',').map<string>((folder: string) => path.resolve(folder)),
-    },
-    {
-      name: '--assetPlugins [list]',
-      description:
-        'Specify any additional asset plugins to be used by the packager by full filepath',
-      parse: (val: string) => val.split(','),
-    },
-    {
-      name: '--assetExts [list]',
-      description:
-        'Specify any additional asset extensions to be used by the packager',
-      parse: (val: string) => val.split(','),
-    },
-    {
-      name: '--sourceExts [list]',
-      description:
-        'Specify any additional source extensions to be used by the packager',
-      parse: (val: string) => val.split(','),
-    },
-    {
-      name: '--platforms [list]',
-      description:
-        'Specify any additional platforms to be used by the packager',
-      parse: (val: string) => val.split(','),
-    },
-    {
-      name: '--providesModuleNodeModules [list]',
```

Comment:
This is a dead option and was never accessed from code. It's also removed from Metro anyway in the recent release.